roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -25,7 +25,7 @@ global
   daemon
   pidfile /var/run/haproxy/haproxy.pid
   tune.bufsize {{ haproxy.bufsize }}
-  stats socket /var/run/haproxy/stats.sock mode 600
+  stats socket /var/run/haproxy/stats.sock mode 644
```

Comment:
Do we want world readable, or do we want to make it group readable by the monitoring group? What's the risk of world readable (really though ANY unwanted user on these hosts is a bad thing).